include/app/tabs/openauto.hpp

```diff
@@ -17,6 +17,7 @@
 
 #include <app/config.hpp>
 #include <app/theme.hpp>
+#include <thread>
```

Comment:
This should be included in the "other" include group (already done in develop so you may have a conflict when you update your branch)